localisation/customizable_localisation/wc_customizable_localisation_artifact_names.txt

```diff
@@ -0,0 +1,1894 @@
+		localisation_key = artifact_jewelry_legendary_prefix_learning_2
+	}
+	text = {
+		trigger = { has_artifact_flag = temp_artifact_prefix_flag_3 }
+		localisation_key = artifact_jewelry_legendary_prefix_learning_3
+	}
+	text = {
+		trigger = { has_artifact_flag = temp_artifact_prefix_flag_4 }
+		localisation_key = artifact_jewelry_legendary_prefix_learning_4
+	}
+	text = {
+		trigger = { has_artifact_flag = temp_artifact_prefix_flag_5 }
+		localisation_key = artifact_jewelry_legendary_prefix_learning_5
+	}
+	text = {
+		trigger = { has_artifact_flag = temp_artifact_prefix_flag_6 }
+		localisation_key = artifact_jewelry_legendary_prefix_learning_6
+	}
+	text = {
+		trigger = { has_artifact_flag = temp_artifact_prefix_flag_7 }
+		localisation_key = artifact_jewelry_legendary_prefix_learning_7
+	}
+	text = {
+		trigger = { has_artifact_flag = temp_artifact_prefix_flag_8 }
+		localisation_key = artifact_jewelry_legendary_prefix_learning_8
+	}
+	text = {
+		trigger = { has_artifact_flag = temp_artifact_prefix_flag_9 }
+		localisation_key = artifact_jewelry_legendary_prefix_learning_9
+	}
+	text = {
+		trigger = { has_artifact_flag = temp_artifact_prefix_flag_10 }
+		localisation_key = artifact_jewelry_legendary_prefix_learning_10
+	}
+	fallback_text = { localisation_key = artifact_jewelry_legendary_prefix_0 }
+}
+
+# Jewelry Uncommon Suffix
+defined_text = {
+	name = GetArtifactJewelryUncommonDiplomacySuffix
+
+	text = {
+		trigger = { has_artifact_flag = temp_artifact_suffix_flag_1 }
+		localisation_key = artifact_jewelry_uncommon_suffix_diplomacy_1
+	}
+	text = {
+		trigger = { has_artifact_flag = temp_artifact_suffix_flag_2 }
+		localisation_key = artifact_jewelry_uncommon_suffix_diplomacy_2
+	}
+	text = {
+		trigger = { has_artifact_flag = temp_artifact_suffix_flag_3 }
+		localisation_key = artifact_jewelry_uncommon_suffix_diplomacy_3
+	}
+	text = {
+		trigger = { has_artifact_flag = temp_artifact_suffix_flag_4 }
+		localisation_key = artifact_jewelry_uncommon_suffix_diplomacy_4
+	}
+	text = {
+		trigger = { has_artifact_flag = temp_artifact_suffix_flag_5 }
+		localisation_key = artifact_jewelry_uncommon_suffix_diplomacy_5
+	}
+	text = {
+		trigger = { has_artifact_flag = temp_artifact_suffix_flag_6 }
+		localisation_key = artifact_jewelry_uncommon_suffix_diplomacy_6
+	}
+	text = {
+		trigger = { has_artifact_flag = temp_artifact_suffix_flag_7 }
+		localisation_key = artifact_jewelry_uncommon_suffix_diplomacy_7
+	}
+	text = {
+		trigger = { has_artifact_flag = temp_artifact_suffix_flag_8 }
+		localisation_key = artifact_jewelry_uncommon_suffix_diplomacy_8
+	}
+	text = {
+		trigger = { has_artifact_flag = temp_artifact_suffix_flag_9 }
+		localisation_key = artifact_jewelry_uncommon_suffix_diplomacy_9
+	}
+	text = {
+		trigger = { has_artifact_flag = temp_artifact_suffix_flag_10 }
+		localisation_key = artifact_jewelry_uncommon_suffix_diplomacy_10
+	}
+	fallback_text = { localisation_key = artifact_jewelry_uncommon_suffix_0 }
```

Comment:
I see that you localized it using `[Nothing]` probably... to make localization empty? But, I want to suggest another way:
```suggestion
	fallback_text = { localisation_key = "" }
```
No need in localization, completely empty, check it.

Reply:
Done, thanks!